server/admin_management/api/app/services/access_control_privilege_service.py

```diff
@@ -61,6 +62,9 @@ def create_access_control_privilege_many(
         fam_user = self.user_service.find_or_create(
             request.user_type_code, request.user_name, requester
         )
+        fam_user = self.user_service.update_user_business_guid(
+            fam_user.user_id, target_user.business_guid,
+            requester)
```

Comment:
Update value for "business_guid" for the target_user after user has found or newly created.